tests/phpunit/tests/functions/isSerialized.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Tests for `is_serialized()`.
+ *
+ * @ticket 53299
+ *
+ * @group functions.php
+ * @covers ::is_serialized
+ */
+class Tests_Functions_IsSerialized extends WP_UnitTestCase {
+
+	/**
+	 * Data provider method for testing `is_serialized()`.
+	 *
+	 * @return array
+	 */
+	public function _is_serialized() {
+		return array(
+
+			// pass array, not a string.
+			array( array(), false ),
```

Comment:
```suggestion
			'an empty array' => array(
				'data'     => array(),
				'expected' => false,
			),
```
- Using dataset keys makes it clearer for testers using the `--testdox` switch for `phpunit` and is therefore more useful than inline comments.
- Using keys for each array element makes it clear what each value refers to. Think: `array( false, false )` - Which is which? Providing keys makes it immediately clear without scrolling, and speeds up writing tests.

This should be done for all of the datasets, such as:
- `'an object'` (for the `new stdClass()` value - this is an object, not a class).
- `'an integer'`
- `'a value that is too short'`
- `'no colon in the second position'`
- `'no trailing semicolon'`
- `'a valid entry'`
- `'an Enum'`